### DISCUSSION TITLE
Does GithubOperator support xcom?

### DISCUSSION THREAD

**Discussion by kimminw00** [apache/airflow#35262] [Q&A] [answered]
When I executed [example DAG](https://github.com/apache/airflow/blob/providers-github/2.4.0/tests/system/providers/github/example_github.py), I couldn't get xcom values from the <code>GithubOperator</code>s. Is there any ways to get the xcom values from <code>GithubOperator</code>?

**Comment by Taragolis** [comment, marked as answer]
I assume that if you tried to use example without change, then only None could be returned into the XCom key "return_value"

`GithubOperator` conditionally return value from method response

https://github.com/apache/airflow/blob/659d94f0ae89f47a7d4b95d6c19ab7f87bd3a60f/airflow/providers/github/operators/github.py#L71-L74

However in example `result_processor` return result of `logging.info` which is always return None

https://github.com/apache/airflow/blob/e9987d50598f70d84cbb2a5d964e21020e81c080/tests/system/providers/github/example_github.py#L96


You could try use instead

```python
result_processor=lambda repo: list(repo.get_tags()), 
```

I'm not familiar with github provider and not sure is it possible directly serialise response of `repo.get_tags()`, so it might required to convert to something which might be serialised to JSON
